lms/services/lti_user.py

```diff
@@ -0,0 +1,56 @@
+from lms.models.lti_user import LTIUser, display_name
+from lms.services import LTIRoleService
+
+
+class LTIUserService:
+    """
+    A service to handle LTIUser.
+
+    LTIUser is the center of many of our authentication schemas.
+    This service handles de/serialization of LTIUsers.
+    """
+
+    def __init__(self, lti_role_service):
+        self._lti_roles_service = lti_role_service
+
+    def from_auth_params(self, application_instance, lti_core_schema) -> LTIUser:
+        """Create an LTIUser from a LTIV11CoreSchema like dict."""
```

Comment:
This description is a bit weird. Do we just mean that the dict contains LTI 1.1 parameters?